content/news/2021/2021-09-10-how-to-supplement-cloudwatch-and-save-your-sanity.md

```diff
@@ -0,0 +1,171 @@
+
+You can and should probably make use of the above guide, but we're also going to wire things up so that any info you see in Cloudwatch that you'd like more details on can be found in Exceptionless, an open-source event monitoring tool. To follow along, you'll need an AWS free tier account, and you'll need to sign up for a free Exceptionless account. 
+
+## What We'll Build
+
+We are going to make this as simple as possible by building a [Lambda](https://aws.amazon.com/lambda/) function and deploying it using [Claudia.js](https://claudiajs.com/). We will connect to that function through Postman or any other rest client (or you can simply use curl from the command line). 
+
+The intent here is not to focus so much on the build-out of the lambda but instead to watch how logging works in Cloudwatch and determine how we can layer in Exceptionless. 
+
+## Creating The Lambda
+
+To get started, you'll need Node.js version 8.0 or above installed on your machine and a good text editor. From the command line, in the directory where you'd like your project to live, run the following command to install Claudia: 
+
+```
+npm install claudia -g
+```
+
+This will install Claudia globally and then you can use it to generate Lambda projects. Since Claudia makes use of your AWS account, it will need to access your AWS credentials. 
+
+Before making sure Claudia has access, let's make sure you have the AWS CLI installed. You can find the [install instructions for your machine here](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html). Once that is installed, run `aws configure`. This will set up your profile with credentials stored locally on your machine.
+
+If you have not created an IAM role in AWS before, [this is a good guide](https://docs.aws.amazon.com/lambda/latest/dg/lambda-intro-execution-role.html) on doing so with permissions specific to Lambda. However, it's important to note that per the docs, the role you specify for Claudia needs the following permissions: 
+
+* AWSLambdaFullAccess is required for all Claudia deployments
+* IAMFullAccess is required if you want Claudia to automatically create execution roles for your Lambda function (recommended for beginners). If you cannot use this role due to corporate policies, or do not want to grant Claudia access to IAM, then create a Lambda execution role manually, and pass the name using --role <ROLE NAME> when executing claudia create.
+* AmazonAPIGatewayAdministrator is required if you want to use Claudia API Builder, Claudia Bot Builder or deploy API Gateway Proxy APIs.
+
+Once you have created your IAM role and have your AWS key and secret, you will need to make sure they are accessible to Claudia. The easiest way to do this is to keep them in the same credentials file as your main AWS creds on your machine. You can access the credentials file and edit it by running the following: 
+
+```
+vim ~/.aws/credentials
```

Comment:
might want to change this to `code ~/.aws/credentials` as no one knows how to exit vim. Or better yet link to documentation for this so it's more cross platform and we don't have to maintain it.

Reply:
Lol this is true